initrd/bin/kexec-unseal-key

```diff
@@ -38,8 +38,15 @@ for tries in 1 2 3; do
 		exit 0
 	fi
 
-	DEBUG $(pcrs)
-	warn "Unable to unseal disk encryption key"
+	warn "Unable to unseal LUKS Disk Unlock Key from TPM"
+	if [ -e /boot/kexec_lukshdr_hash.txt ] && [ -e /tmp/luksDump.txt ]; then
+		if ! cmp -s /boot/kexec_lukshdr_hash.txt /tmp/luksDump.txt > /dev/null 2>&1; then
+			warn "Encrypted disk keys(s) have changed since sealed in TPM Disk Unlock Key. You might want to investigate."
```

Comment:
I like the precision of specifying that they have not been changed relative to a specific point in time (sealing the TPM Disk Unlock Key).  But IMO "You might want to investigate." is unnecessarily vague, we can say _why_ they may want to investigate.

So I'd suggest:
`Encrypted disk keys have changed since the TPM Disk Unlock Key was sealed. If you did not make this change, the disk may be compromised.`